cmd/helm/uninstall.go

```diff
@@ -27,8 +27,9 @@ import (
 )
 
 const uninstallDesc = `
-This command takes a release name, and then uninstalls the release from Kubernetes.
-It removes all of the resources associated with the last release of the chart.
+This command takes a release name and uninstalls the release from Kubernetes.
+It removes all of the resources associated with the last release of the chart
+as well as the release history, freeing it up for future use.
```

Comment:
Would this read better considering that the Helm history might be stored outside of Kubernetes at some stage?

This command takes a release name and uninstalls the release.
It removes all resources from Kubernetes associated with the last release of the chart. It removes the release history as well.

Reply:
> Would this read better considering that the Helm history might be stored outside of Kubernetes at some stage?

When would the Helm history be stored outside of the cluster?